app/views/system_admin/duplicate_hesa_trainees/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= render PageTitle::View.new(i18n_key: "duplicate_hesa_trainees.index") %>
+
+<div class="govuk-grid-row">
+  <div class ="govuk-grid-column-full">
+    <%= render "system_admin/tab_nav" %>
+
```

Comment:
Can we display a title and have the the total number of duplicate records in parentheses after the header, i.e. Duplicate HESA trainees (25), as per the ticket?